src/mavis/interval.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, start: int, end: Optional[int] = None, freq: int = 1, number_
 
         self.start = self.number_type(self.start)
         self.end = self.number_type(self.end)
+        if self.start == 0 and self.end == 1:
+            self.start = 1
```

Comment:
can you add an inline comment here that points to the related github issue? I think we might look at this in the future and forget why it is doing this